package.json

```diff
@@ -138,11 +138,11 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^22.13.0 || ^24.10.0",
+    "node": "^22.13.0 || ^24.11.0",
```

Comment:
this is what I consider a breaking change, but this time it's only cosmetic, so could also not be done to be a simple feature release